modules/update-service-perform-update.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// * updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc
+
+:_content-type: PROCEDURE
+[id="update-service-perform-update"]
+= Updating the disconnected cluster with the OpenShift Update Service
```

Comment:
One question - I don't know if this module is reused elsewhere but considering that this is a single step procedure with links only, would it make more sense to move this content to the assembly?

Reply:
That's a good question. This assembly is definitely going to be the only place this module will ever be included in. I'll ask around to see if it's okay to include the procedure directly in the assembly, or if that's a bad practice per modular documentation guidelines.

I would definitely prefer to include the procedure in the assembly, that way I can put xrefs in the bullet list instead of having to add an additional resources section after it.

Reply:
I think if you include the content in the assembly, it would not be in the form of an actual procedure per se, but rather an unordered list. There are examples of this in the assembly. Something like the `Mirroring the {product-title} image repository` section maybe? I will leave it up to you to decide.